js/ui/list/ui.list.edit.decorator.reorder.js

```diff
@@ -43,11 +43,14 @@ registerDecorator(
         },
 
         _dragTemplate: function(e) {
+            const direction = this.option('rtlEnabled') ? 'rtl' : 'ltr';
+
             return $(e.itemElement)
                 .clone()
                 .width($(e.itemElement).width())
                 .addClass(REORDERING_ITEM_GHOST_CLASS)
-                .addClass(STATE_HOVER_CLASS);
+                .addClass(STATE_HOVER_CLASS)
+                .css('direction', direction);
```

Comment:
add `const RTL_CLASS = 'dx-rtl';`
```suggestion
                .toggleClass(RTL_CLASS, this.option('rtlEnabled'));
```